app/assets/javascripts/samples/sample_datatable.js

```diff
@@ -736,14 +736,21 @@ function createTdElement(content) {
 
 /**
  * Creates select dropdown for sample type
- * @param data List of sample types
- * @param selected Selected sample type id
+ * @param {Object[]} data List of sample types
+ * @param {number} selected Selected sample type id
+ * @return {Object} select dropdown
  */
 function createSampleTypeSelect(data, selected) {
+  selected = _.isUndefined(selected) ? 1 : selected + 1;
+
   var $selectType = $('<select></select>')
     .attr('name', 'sample_type_id').addClass('show-tick');
 
-  var $option = $('<option></option>')
+  var $option = $("<option href='/organizations/1/sample_types'></option>")
```

Comment:
Do you hardcoded the link that links always to the first organization ??????? svašta